pkg/services/govmomi/util.go

```diff
@@ -366,7 +366,7 @@ func waitForMacAddresses(ctx *virtualMachineContext) error {
 func getMacAddresses(ctx *virtualMachineContext) ([]string, map[string]int, map[int]string, error) {
 	var (
 		vm                   mo.VirtualMachine
-		macAddresses         []string
+		macAddresses         = make([]string, 0, len(vm.Config.Hardware.Device))
```

Comment:
```suggestion
		macAddresses         = make([]string, 0)
```
I think len is always 0?

Reply:
Yes, of course, I missed that.